bazel/ray_java.bzl

```diff
@@ -0,0 +1,1577 @@
+# The following dependencies were calculated from:
```

Comment:
If we don't auto-generate this file in this PR, let's leave a `TODO` comment here to take a note.

Reply:
So do we truely need all these dependencies? It seems to me that this file could be slimmed down a lot, for example GRPC should not be needed to build this.